app/points/calculate-community-tier-workload-points.js

```diff
@@ -0,0 +1,15 @@
+const calculatePointsForTier = require('../../app/points/calculate-points-for-tier')
+
+module.exports = function (communityTiers, caseTypeWeightings, commTierCPEnabled, commTier3AEnabled) {
+  var communityWorkloadPoints = 0
+  communityTiers.getTierCountsAsList().forEach(function (tier) {
+    communityWorkloadPoints += calculatePointsForTier(tier.tierCount, tier.points, caseTypeWeightings)
+  }, this)
+  if (commTierCPEnabled) {
+    communityWorkloadPoints += calculatePointsForTier(communityTiers.a1.tierCount, communityTiers.a1.points, caseTypeWeightings.commTierCPWeighting)
```

Comment:
Yep, you are right. I was more wanting to use the method calculatedWeightedPoints. Got them confused.